presto-main/src/main/java/com/facebook/presto/sql/planner/plan/AggregationNode.java

```diff
@@ -159,6 +160,16 @@ public double getConfidence()
         return groupingSets;
     }
 
+    @JsonProperty("groupingSetsIntersection")
```

Comment:
Why does this need to be serializable?

Also, I might rename this to `getCommonGroupingColumns`